src/smclarify/bias/metrics/pretraining.py

```diff
@@ -108,11 +108,11 @@ def JS(label: pd.Series, sensitive_facet_index: pd.Series) -> float:
     require(sensitive_facet_index.dtype == bool, "sensitive_facet_index must be of type bool")
     xs_a = label[~sensitive_facet_index]
     xs_d = label[sensitive_facet_index]
-    (Pa, Pd, P) = pdfs_aligned_nonzero(xs_a, xs_d, label)
+    (Pa, Pd, _) = pdfs_aligned_nonzero(xs_a, xs_d, label)
+    P = 1 / 2 * (Pa + Pd)
     if len(Pa) == 0 or len(Pd) == 0 or len(P) == 0:
```

Comment:
We don't need to pass `label` to the function anymore because `xs_a` and `xa_d` together have all values of `label`. Code like below should work fine,

```
..
(Pa, Pd) = pdfs_aligned_nonzero(xs_a, xs_d)
P = 0.5 * (Pa + Pd)
if len(Pa) == 0 or len(Pd) == 0:
...